app/routes/index.js

```diff
@@ -132,7 +146,7 @@ router.post('/run-process', async (req, res) => {
         return
       }
 
-      if (!request || !request.inputs || !request.outputs) {
+      if (!request || !request.inputs || !request.outputs || (await containsInvalidMembershipOwners(request.outputs))) {
         logger.trace(`Request missing input and/or outputs`)
         res.status(400).json({ message: `Request missing input and/or outputs` })
```

Comment:
The error message here is meybe now a bit confusing. Consider splitting the `if` into multiple statements so you can produce a better error on the invalid member case